docs/jwst/refpix/description.rst

```diff
@@ -108,7 +108,7 @@ NIR Data
 ++++++++
 
 For single amplifier readout (NOUTPUTS keyword = 1):
-----------------------------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Yep, this appears to fix the section headings nicely, as evidence in the doc build from the PR CI.